doc/manual/parallel-computing.rst

```diff
@@ -1013,7 +1013,8 @@ using the command :obj:`Threads.nthreads()`::
     1
 
 The number of threads Julia starts up with is controlled by an environment variable 
-called ``JULIA_NUM_THREADS``. Now, let's start up Julia with 4 threads::
+called ``JULIA_NUM_THREADS``. (Note that if you're using a C shell, you should use the keyword 
+``set`` instead of ``export``). Now, let's start up Julia with 4 threads::
```

Comment:
Unless you're using a shell like Cygwin. Can I point to some kind of online guide to change environment variables on Windows? Like [this](http://www.computerhope.com/issues/ch000549.htm)?